pkg/common/genericSet.go

```diff
@@ -51,3 +51,12 @@ func (s GenericSet[T]) IsIntersect(s2 GenericSet[T]) bool {
 	}
 	return false
 }
+
+func Contains[T comparable](s []T, e T) bool {
+	for _, a := range s {
+		if a == e {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
go 1.21 has this functionality, see: https://pkg.go.dev/slices#Contains 
better use `slices.Contains()` instead of implementing it here

Reply:
indeed! done.